.github/workflows/style.yml

```diff
@@ -1,15 +1,12 @@
 name: Style Checker
 
-# Runs my simple style(9) checker on any pushes or pull requests.  It could be
-# optimized by fetching the pull request head branch back to main revisions and
-# running on that. That would reduce the run time from 3-4 minutes down to 30-40
-# seconds. Getting the right series of clone + fetches to get that iteratively
-# is proving elusive, so optimizations welcome.
+# Runs my simple style(9) checker on pull requests.
 
 on:
   pull_request: # maybe pull_request_target
     branches: [ main ]
     types: [ opened, reopened, edited, synchronize ]
+    paths: [ '**.S', '**.c', '**.cc', '**.h', '**.hh' ]
```

Comment:
Needs **.cpp and **.hpp as well.
Though the style script also needs an update for those two. I'll take care of that and have it stop commenting on perl and python.